signer-service/src/api/controllers/brla.controller.ts

```diff
@@ -0,0 +1,164 @@
+import { Request, Response } from 'express';
+import { BrlaApiService } from '../services/brla/brlaApiService';
+import { RegisterSubaccountPayload, TriggerOfframpRequest } from '../services/brla/types';
+import { eventPoller } from '../..';
+
+export const getBrlaUser = async (
+  req: Request<{}, {}, {}, { taxId: string; pixId: string }>,
+  res: Response,
+): Promise<void> => {
+  try {
+    const { taxId, pixId } = req.query;
+
+    if (!taxId || !pixId) {
+      res.status(400).json({ error: 'Missing taxId or pixId query parameters' });
+      return;
+    }
+
+    // TODO how to check that pixId is valid, as a later offramp will get stuck if it's not valid..
+
+    const brlaApiService = BrlaApiService.getInstance();
+    const subaccount = await brlaApiService.getSubaccount(taxId);
+    if (!subaccount) {
+      res.status(404).json({ error: 'Subaccount not found' });
+      return;
+    }
+    if (subaccount.kyc.level !== 1) {
```

Comment:
It can be greater or equal to 1 as far as I understand. 
```suggestion
    if (subaccount.kyc.level < 1) {
```

Reply:
Good catch.